JWT.js

```diff
@@ -0,0 +1,61 @@
+const jwt = require("jsonwebtoken");
```

Comment:
feel free to convert this file to `jwtService.js` and export class instance, not signle functinos